src/three/plugins/GoogleCloudAuthPlugin.js

```diff
@@ -46,6 +47,15 @@ export class GoogleCloudAuthPlugin {
 
 	init( tiles ) {
 
+		if ( this.endpointURL === null ) {
+
+			this.endpointURL = 'https://tile.googleapis.com/v1/3dtiles/root.json';
+
+		}
+
+		tiles.rootURL = this.endpointURL;
+
```

Comment:
I think we should only assign this if the user hasn't set root url when creating `TilesRenderer` - I don't think there's a lot of value in taking a full URL through the tiles constructor _and_ the plugin options. The plugin should just assign a default if the user hasn't provided one.

Reply:
Agreed. 

Will replace with 
```js

	init( tiles ) {

		if ( tiles.url === null ) {

			tiles.url = 'https://tile.googleapis.com/v1/3dtiles/root.json';

		}
        //...
```